homeassistant/components/owntracks/messages.py

```diff
@@ -39,11 +39,15 @@ def _parse_topic(topic, subscribe_topic):
     Async friendly.
     """
     subscription = subscribe_topic.split("/")
-    try:
-        user_index = subscription.index("#")
-    except ValueError:
+
+    user_index = -1
+    for i, s in enumerate(subscription):
```

Comment:
Please rename `i` and `s` to something more self-explanatory and conform to snake_case naming style.